x-pack/plugins/apm/server/routes/register_routes/index.ts

```diff
@@ -116,9 +121,22 @@ export function registerRoutes({
           // cleanup
           inspectableEsQueriesMap.delete(request);
 
+          if (!options.disableTelemetry && telemetryUsageCounter) {
+            telemetryUsageCounter.incrementCounter({
+              counterName: `${method}_${pathname}`,
```

Comment:
Like @smith I also prefer spaces instead of underscores if possible:

```suggestion
              counterName: `${method} ${pathname}`,
```